### DISCUSSION TITLE
Help with Docker Deployment

### DISCUSSION THREAD

**Discussion by burjuyz** [oussamatn/radio-player#11] [Q&A] [answered]
I deployed the player in the same server as AzuraCast Core when i try to connect to subdomain player.bbz.ru it sends me to login page.
When i use http://player.bbz.ru:8080/ it works.
How to make in azuracast redirect to player.bbz.ru, without port.

**Comment by joeyboli** [comment, marked as answer]
Same thing happened to me. Just use the local method
